lib/portlayer/exec/container_cache.go

```diff
@@ -48,11 +61,35 @@ func NewContainerCache() {
 func (conCache *containerCache) Container(idOrRef string) *Container {
 	conCache.m.RLock()
 	defer conCache.m.RUnlock()
-	// find by id
+	// find by id or moref
 	container := conCache.cache[idOrRef]
 	return container
 }
 
+// return an array of pointers to the containers in the cache
+// TODO: implement filtering here?
+func (conCache *containerCache) Containers(allStates *bool) []*Container {
```

Comment:
sure we could put it there -- I placed here so that others could use if desired -- in a near term change the cache will be moved out of exec and accessible by other areas.  So the thought being those other areas might need to filter -- this would provide them a standard way to do that...